### DISCUSSION TITLE
UI is too complecated and unintuiteve

### DISCUSSION THREAD

**Discussion by hemangjoshi37a** [FreeCAD/FreeCAD#18614] [Code contribution discussions]
I am having hard time learning freecad because of it's hard UI.

If any developer can integrate this UI implementation it will be a blessing : https://github.com/HakanSeven12/Modern-UI

https://github.com/geolta/FreeCAD-Ribbon

**Comment by luzpaz** [comment]
You won't get traction here. forum.freecad.org is the best place to discuss. Remember not everyone shares your opinion. People have been using FreeCAD for 20 years and don't prefer Solidworks/Fusion UI/UX over FreeCAD. That is why we have addons.

**Reply by hemangjoshi37a** [reply]
Got it
